webpack.config.js

```diff
@@ -50,7 +50,8 @@ module.exports = async (env, options) => {
     },
     plugins: [
       new CleanWebpackPlugin({
-        cleanOnceBeforeBuildPatterns: dev ? [] : ["**/*"]
+        cleanOnceBeforeBuildPatterns: dev ? [] : ["**/*"],
+        cleanAfterEveryBuildPatterns: ["!**/*"]
```

Comment:
Should the cleanAfterEveryBuildPatterns be different between dev and prod?  e.g. `dev ? [!**/*] : []`

Reply:
Sure, it makes sense to be that way if the prod is already working. 
Just pushed this fix